### DISCUSSION TITLE
docs question

### DISCUSSION THREAD

**Discussion by gisraelsplunk** [reduxjs/redux#4596] [Help]
Hi i am following the tutorial and it says this here `We have a file named features/counter/counterSlice.js that exports a reducer function for the counter logic. We can import that counterReducer function here, and include it when we create the store.

` here's link https://redux.js.org/tutorials/essentials/part-2-app-structure but I don't see anything called counterReducer in counterSlice.js
can someone explain this?

**Comment by markerikson** [comment]
It's normal to do a default export of the reducer function from a slice file, and named exports of the actions:

```ts
const counterSlice = createSlice({ /*   */})

export {increment, decrement} = counterSlice.actions;

export default counterSlice.reducer;
```

When you do the default _import_, you can give that variable any name you want:

```ts
import AnyNameYouWantHere from "../features/counter/counterSlice";
```

But normally you'd call it something like `thatSliceReducer`:

```ts
import counterReducer from "../features/counter/counterSlice";
```
